src/main/cpp/mechanisms/noteManager/generated/noteManagerGen.cpp

```diff
@@ -558,6 +549,7 @@ void noteManagerGen::CreatepracticeBot9999()
 	placerMidSensor = new DragonDigitalInput ( "placerMidSensor",RobotElementNames::DIGITAL_INPUT_USAGE::NOTE_MANAGER_PLACER_MID_SENSOR,6,true,units::time::second_t ( 0 ) );
 	placerOutSensor = new DragonDigitalInput ( "placerOutSensor",RobotElementNames::DIGITAL_INPUT_USAGE::NOTE_MANAGER_PLACER_OUT_SENSOR,7,true,units::time::second_t ( 0 ) );
 
+	launcherAngleEncoder = new DragonCanCoder ( "launcherAngleEncoder",RobotElementNames::CANCODER_USAGE::NOTE_MANAGER_LAUNCHER_ANGLE_ENCODER,0,"rio",0,false );
```

Comment:
Unless we are adding a CanCoder on the practice bot, this will not work. I assume that you made the decision to concentrate on the CompBot

Reply:
Correct, we aren't really using the practice bot currently and I believe it will be updated for offseason if we want to use it